packages/fast-usdc/src/fast-usdc.contract.js

```diff
@@ -20,30 +23,33 @@ const trace = makeTracer('FastUsdc');
  * @import {OrchestrationPowers, OrchestrationTools} from '@agoric/orchestration/src/utils/start-helper.js';
  * @import {Zone} from '@agoric/zone';
  * @import {OperatorKit} from './exos/operator-kit.js';
- * @import {CctpTxEvidence} from './types.js';
+ * @import {CctpTxEvidence, FeeConfig} from './types.js';
  */
 
 /**
  * @typedef {{
- *   poolFee: Amount<'nat'>;
- *   contractFee: Amount<'nat'>;
  *   usdcDenom: Denom;
  * }} FastUsdcTerms
  */
-const NatAmountShape = { brand: BrandShape, value: M.nat() };
-export const meta = {
+
+export const meta = /** @type {ContractMeta<FastUsdcSF>} */ ({
```

Comment:
how is this better than the existing convention?
```suggestion
/** @type {ContractMeta<typeof start>} */
```